### DISCUSSION TITLE
Project Requests

### DISCUSSION THREAD

**Discussion by diStyApps** [diStyApps/seait#11] [Announcements]
_The world is changed. I feel it in the water. I feel it in the earth. I smell it in the air. Much that once was is lost, for none now live who remember it._ - Galadriel, J.R.R. Tolkien.

The pace of change is accelerating, with numerous new projects emerging constantly. Even though I stay well-updated, it's easy to lose track of everything. That's why I need your help to stay current and to know which projects you'd like to 'SEAIT' in action.

I will do my best to add your requested project, but I guess some will be harder to add and some may be impossible. However, with your support, the goal of fully automating "SEAIT" can be achieved.

So go ahead and make a request!

**Comment by Flashwalker** [comment]
How about Linux installer or at least install instructions?

**Reply by diStyApps** [reply]
> How about Linux installer or at least install instructions?

Ill add something soon.

**Comment by naitedj** [comment]
please revert to an older version.

**Reply by diStyApps** [reply]
Why what is not working for you?

**Comment by thijsi123** [comment]
An option to stop a server would be appreciated.

**Comment by toon-geek** [comment]
RVC WebUI

**Comment by Latnok** [comment]
Thank you for your work.
Add this tool, please https://github.com/XpycT/MagicPrompt

**Comment by Shouri14** [comment]
Amazing work, really simple and useful for multiple projects. Thank you for your effort and time.

Was wondering if LocalGPT can be added ? (https://github.com/PromtEngineer/localGPT)

Thank you.

**Comment by Fearganainim** [comment]
Vision Crafter. There is a new Animatediff model can you add it?
